src/main/reactjs/src/components/clerkTranslator/add/AddAuthorisation.tsx

```diff
@@ -186,6 +195,8 @@ export const AddAuthorisation = ({
 
   const testIdPrefix = 'add-authorisation-field';
 
+  useNavigationProtection(isAuthorisationDataChanged);
```

Comment:
Toimisi eri tavoin kuin muualla, mutta olisko täällä kätevämpää antaa vain parametrina: `authorisation !== newAuthorisation` ja näin ei tarvittaisi säilyttää myöskään statessa tietoa tuon muuttumisesta.

Näemmä tuo vertailu ei tuollaisenaan toimi suoraan, mutta jos vertailisi objekteja yleisesti.

Reply:
Tuossa on se ikävyys, että useEffectillä lisätään `authorisation`iin tempId, jolloin nämä oliot eivät ikinä mätsää. Katselin vähän vaihtoehtoja tuohon, mutta rumalta näytti nekin.

Reply:
Jes. No mut mennään tällä nykyisellä ratkaisulla tämän osalta sitten.